packages/@react-spectrum/numberfield/docs/NumberField.mdx

```diff
@@ -0,0 +1,218 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/numberfield';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/numberfield/package.json';
+
+```jsx import
+import {NumberField} from '@react-spectrum/numberfield';
+import {Flex} from '@react-spectrum/layout';
+```
+
+---
+category: Forms
+keywords: [number field, input]
+---
+
+# NumberField
+
+<p>{docs.exports.NumberField.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['NumberField']}
+  sourceData={[]} />
+
+## Example
+
+```tsx example
+<NumberField label="Name" />
+```
+
+## Value
+
+A NumberField's `value` is empty by default, but an initial, uncontrolled, value can be provided using the `defaultValue` prop.
+Alternatively, a controlled value can be provided using the `value` prop.
+
+```tsx example
+function Example() {
+  let [value, setValue] = React.useState(15);
+
+  return (
+    <Flex gap="size-150" wrap>
+      <NumberField
+        label="Cookies (Uncontrolled)"
+        defaultValue={15} />
+
+      <NumberField
+        label="Cookies (Controlled)"
+        value={value}
+        onChange={setValue} />
+    </Flex>
+  );
+}
+```
+
+## Steps
+
+A NumberField can have a specified step size, as well as a min and max value. The steps will always start at the `minValue` or `0`
+and will not exceed the min or the max if they are specified.
```

Comment:
I think this needs more description to explain the nuances of the step behavior. Maybe reuse the description from the aria docs? https://github.com/adobe/react-spectrum/pull/1523/files#diff-baa7d39d9b1465a7d8ce5b1760066982303b4f8a93fc797af787879bef41bfb4R273-R282